lib/compat/wordpress-6.7/formatting.php

```diff
@@ -0,0 +1,44 @@
+<?php
+/**
+ * Extensions to the WordPress Formatting API.
+ * src/wp-includes/formatting.php
+ *
+ * @package gutenberg
+ */
+
+/**
+ * Given a numeric value, returns a rounded a valid CSS <number> as a string.
+ * Negative zero values, e.g., `-0.0`, will return "0".
+ * See https://developer.mozilla.org/en-US/docs/Web/CSS/number
+ * Normalizes LM_NUMERIC locales with non-dot decimal point, e.g., "1,234.56" to "1234.56".
+ * The above is relevant to PHP < 8, whose float to string casting is locale-dependent.
+ * See https://php.watch/versions/8.0/float-to-string-locale-independent
+ *
+ * Usage:
+ * Call just before constructing the final output of your CSS rules, and after any number-based calculation.
+ *
+ * $padding_rounded = gutenberg_round_css_value( 20.3566 + 5, array( 'decimal_places' => 1 ) );
+ * echo "padding: {$padding_rounded}px;" // "padding: 25.4px;"
+ *
+ * @param int|string|float $value A CSS <number> data type.
+ * @param array            $options {
+ *     Optional. An array of options. Default empty array.
+ * }
+ * @return string A rounded CSS number.
+ */
+function gutenberg_round_css_value( $value, $options = array() ) {
```

Comment:
what about `gutenberg_rounded_css_value()` to focus more on the output than the process?

alternatively, just `gutenberg_format_css_number()` or `gutenberg_css_number()` or `gutenberg_to_css_number()` all speak more directly to the semantic this is attempting to provide: take a PHP number and return a CSS number.